src/Nest/CommonAbstractions/ConnectionSettings/ConnectionSettingsBase.cs

```diff
@@ -172,9 +173,10 @@ public TConnectionSettings MapDefaultTypeIndices(Action<FluentDictionary<Type, s
 		}
 
 		/// <summary>
-		/// Specify the default type names for a given POCO type. 
+		/// Specify the default type names for a given POCO type.
 		/// Takes precedence over the global <see cref="DefaultTypeNameInferrer"/>
 		/// </summary>
+		[Obsolete("Removed in 6.0.")]
```

Comment:
I'm inclined not to `Obsolete` this as it is likely to be noisy for a lot of users, since it is a commonly used method. Perhaps an XML comment instead?